.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: dapr-rust-sdk
+
+on: [push, pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_TOKEN: ${{ secrets.CRATES_IO_TOKEN }}
+
+jobs:
+  lint:
+    name: Lint 
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: cargo fmt
+      run: cargo fmt -- --check --color ${{ env.CARGO_TERM_COLOR }}
+    - name: cargo clippy
+      run: cargo clippy -- -W warnings
+
+
+  build:
+    name: Build 
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build
+    - name: Build examples
+      run: cargo build --examples
+    - name: Run Tests
+      run: cargo test --all-targets
+   
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    needs: [lint, build]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
I would publish the cargo package only for tag push. Please see the below command:

https://github.com/dapr/cli/blob/51bbd0a1715eb00a2cc6e3dd2518a42336418389/.github/workflows/dapr_cli.yaml#L89

So the release process (v0.2.0-alpha.0) will be 
1. create release-* branch
2. push v* tag with v0.2.0-alpha.0 

Please change this workflow to publish cargo package when it is version tag push.

Reply:
Sounds good. This workflow makes more sense.

Reply:
@gdhuper any updates ?

Reply:
Apologies for delay. Will try to wrap this up end of today.